src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -945,11 +951,11 @@ private LoadFlowParametersEntity createParametersEntity(LoadFlowParametersValues
     }
 
     private void deleteLoadFlowResult(UUID studyUuid) {
-        List<UUID> loadFlowResultUuids = networkModificationTreeService.getLoadFlowResultUuids(studyUuid);
+        List<UUID> loadFlowResultUuids = networkModificationTreeService.getComputationResultUuids(studyUuid, LOAD_FLOW);
```

Comment:
I don't want this PR to become a nightmare but see this delete*Result could be factorized as well :smile:

Reply:
Are you sure ? I only found one 'delete*Result' (this one) and it doesn't seem to be used at all. Technically those in the Services could be  refactored (by creating a parent class for ShortCircuitService etc) but I am not sure that I have the will to start this.

Reply:
Well it seems that this method isn't used anymore then maybe just remove it ?

Reply:
I dared not but your encouragement decided me ! Done.